specification/versioning.md

```diff
@@ -77,3 +77,35 @@ until the end of that component’s existence:
   MUST be provided as the latest `PATCH` version for the latest `MINOR` version
   of the latest `MAJOR` and SHOULD NOT be provided for previous `PATCH` or
   `MINOR` releases.
+
+## Release life cycle management
+
+Each repository `MUST` use [GitHub releases mechanism](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)
+to create a versioned snapshot of components and features
+delivered to the users. If the release is also distributed via other
+distribution channels (e.g. CDN, Registries, AppStores) it `MUST` state
+its life cycle status according to following rules by any means available
+in the given software distribution solution.
+
+Each release `MUST` clearly state the life cycle
+status and described as: Supported, Deprecated, End of Support.
```

Comment:
If we use "Obsolete" instead of "Deprecated" then we would have distinct status names for "Maturity" and "Releases". What do you think?

Reply:
I think "Obsolete" is too negative and implies that the thing no longer works, whereas something Deprecated can still work while not being the favored choice.